vkcs/provider_test.go

```diff
@@ -16,17 +18,34 @@ import (
 var (
 	osFlavorID         = os.Getenv("OS_FLAVOR_ID")
 	osNewFlavorID      = os.Getenv("OS_NEW_FLAVOR_ID")
+	osFlavorName       = os.Getenv("OS_FLAVOR_NAME")
+	osNewFlavorName    = os.Getenv("OS_NEW_FLAVOR_NAME")
+	osImageName        = os.Getenv("OS_IMAGE_NAME")
 	osNetworkID        = os.Getenv("OS_NETWORK_ID")
 	osRegionName       = os.Getenv("OS_REGION_NAME")
-	osPoolName         = os.Getenv("OS_POOL_NAME")
 	osProjectID        = os.Getenv("OS_PROJECT_ID")
-	osExtGwID          = os.Getenv("OS_EXTGW_ID")
-	osPrivateDNSDomain = os.Getenv("OS_PRIVATE_DNS_DOMAIN")
+	osExtNetName       = os.Getenv("OS_EXT_NET_NAME")
 	osKeypairName      = os.Getenv("OS_KEYPAIR_NAME")
 	clusterTemplateID  = os.Getenv("CLUSTER_TEMPLATE_ID")
 	osSubnetworkID     = os.Getenv("OS_SUBNETWORK_ID")
```

Comment:
could we group kubernetes only related vars at the end of the list with a comment?

Reply:
Rearranged